kuiper-game/Systems/Events/SetCourseEvent.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Linq;
+using Kuiper.Services;
+
+namespace Kuiper.Systems.Events
+{
+    public class SetCourseEvent : IEvent
```

Comment:
First actual event. It's put into the GameEvent list when a course is set and will be executed when the ship arrives at it's destination.